domain/build.gradle

```diff
@@ -0,0 +1,11 @@
+apply plugin: 'kotlin'
+
+apply from: './gradleFiles/dependenciesDomain.gradle'
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation dependenciesDomain.kotlin
+}
+
+sourceCompatibility = "7"
+targetCompatibility = "7"
```

Comment:
@OscarGonzalezLesaga sourceCompatibility and targetCompatibility are necessary?

Reply:
I'm going to study it

Reply:
I read about it and it's for java compatibility, domain is a full kotlin independent module so I'm going to delete it. Thanks ;)